Agda.cabal

```diff
@@ -506,7 +505,6 @@ library
                     Agda.Utils.Graph.AdjacencyMap.Unidirectional
                     Agda.Utils.Graph.TopSort
                     Agda.Utils.Hash
-                    Agda.Utils.HashMap
```

Comment:
Not sure the removal of Agda.Utils.HashMap is warranted...  May create extra work for users of Agda as a library (also the reason why travis failed in the api test).  Such a change certainly would require an entry in the CHANGELOG to tell users how to migrate.

Reply:
Sure, I'll add this change to CHANGELOG. `Agda.Utils.HashMap` only re-exports `Data.HashMap.Strict`, so it is pretty easy to migrate.